tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -33,6 +34,23 @@ const snackbar = () => $('#snackbar.active .snackbar-message');
 const snackbarMessage = async () => (await $('#snackbar.active .snackbar-message')).getText();
 const snackbarAction = () => $('#snackbar.active .snackbar-action');
 
+//Hamburguer menu
+//User settings
+const USER_SETTINGS = '=User settings';
+const UPDATE_PASSWORD = '=Update password';
```

Comment:
Here it can user this selector:  `.user .configuration.page i.fa-key`
But maybe adding a `test-id` attribute is nicer for all these